wayat/frontend/lib/features/contacts/pages/contacts_page/contacts_page.dart

```diff
@@ -32,7 +34,11 @@ class ContactsPage extends StatelessWidget {
   Expanded contactsPageContent() {
     return Expanded(
         child: AutoTabsRouter.tabBar(
-      routes: [FriendsRoute(), RequestsRoute(), SuggestionsRoute()],
+      routes: [
+        FriendsRoute(), 
+        RequestsRoute(), 
+        if (!PlatformService().isWeb) SuggestionsRoute()
```

Comment:
In this file either add PlatformService as a field and pass it as optional parameter to the component, or as optional parameter to each of the methods.